packages/docs/src/scripts/components/Footer.jsx

```diff
@@ -32,63 +32,102 @@ function renderLinks(links) {
-            <h2 className="ds-h4">Find an issue or have a feature request?</h2>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--2">
             <a
-              className="ds-c-button ds-c-button--primary"
-              href={githubUrl('issues')}
+              className="ds-c-button"
+              href={githubUrl(
+                'issues/new?assignees=&labels=Bug&template=bug_report.md&title='
+              )}
               target="_blank"
             >
-              Ask questions on GitHub
+              Report a bug
             </a>
-            <p>
-              <a href="https://forms.cms.gov/cms-wds-design-system-contact-form/responses/new">
-                Contact the CMSDS team
-              </a>
-            </p>
-            <p className="ds-text ds-u-color--primary-darkest ds-u-font-size--small ds-u-margin-top--3 ds-u-measure--base">
-              A federal government website managed by the Centers for Medicare &
-              Medicaid Services 7500 Security Boulevard, Baltimore, MD 21124
-            </p>
-          </article>
-          <article className="ds-l-col--12 ds-l-sm-col--6 ds-l-lg-col--4 ds-u-margin-bottom--4">
-            <h2 className="ds-h4">CMS &amp; HHS Websites</h2>
-            <ul className="ds-c-list ds-c-list--bare ds-u-font-size--small">
+          </div>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--2">
+            <a
+              className="ds-c-button"
+              href={githubUrl(
+                'issues/new?assignees=&labels=&template=general-issue.md&title='
+              )}
+              target="_blank"
+            >
+              Request a change
+            </a>
+          </div>
+          <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4">
+            <a
+              className="ds-c-button"
+              href={githubUrl(
+                'issues/new?assignees=&labels=proposal%2Ffeature+request&template=propose-a-new-item-for-the-cms-design-system.md&title='
+              )}
+              target="_blank"
+            >
+              Propose something new
+            </a>
+          </div>
+        </div>
+      </section>
+      <section className="ds-base--inverse">
+        <div className="ds-l-container ds-u-padding-y--4">
+          <div className="ds-l-row">
+            <div className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--4 ds-u-margin-top--2">
+              <h6 className="ds-h4">Contact Us</h6>
+              <p className="ds-u-font-size--small">
+                Need help with the CMS Design System?
+                <br />
+                <a href="https://forms.cms.gov/cms-wds-design-system-contact-form/responses/new">
+                  Drop us a line{' '}
+                </a>
+              </p>
+            </div>
+            <dl className="ds-l-col--12 ds-l-sm-col--4 ds-l-lg-col--4 ds-u-margin-bottom--4">
```

Comment:
I'm looking at the `ds-u-margin-bottom--4` styling for these `dl` elements and I'm also wondering if we should only have the margin present when the lists are collapsed on smaller breakpoints. I know that this is the existing style, but you can see in this screenshot there's quite a bit of margin now that that the address is moved to the bottom

https://gyazo.com/d100b47b8d66bdaddfa4b11b5207a089

Reply:
I've updated the padding/margin to better accomodate for the spacing